packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -298,6 +304,16 @@ export default class AgoraIpOrderCtrl {
   }
 
   redirectToPaymentPage() {
+    const { params } = this.model;
+    const serviceType = params.selectedService.type;
+    const offerPlanCode = params.selectedOffer.planCode;
+    const quantity = params.selectedQuantity || 1;
```

Comment:
If `selectedQuantity` can be 0, better to use `nullish coalescing`

Reply:
It is always 1 or above